src/Todo.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './Todo.css';
+import TodoItem from './TodoItems';
+
+// eslint-disable-next-line react/prop-types
+function TodoList({ todos }) {
+  return (
+    <table>
+      <thead>
+        <tr>
+          <th>ID</th>
+          <th>Title</th>
+          <th>Completed</th>
+          <th>User</th>
+        </tr>
+      </thead>
+      {todos.map(i => (<TodoItem todo={i} key={i.id} />
```

Comment:
Что такое i? Давай осмысленные имена переменным
```suggestion
      {todos.map(item => (<TodoItem todo={item} key={item.id} />
```